ci/scripts/single-node-utils.sh

```diff
@@ -12,6 +12,12 @@ export PREFIX_LOG=$RW_PREFIX/log
 start_single_node() {
   mkdir -p "$HOME/.risingwave/state_store"
   mkdir -p "$HOME/.risingwave/meta_store"
+  mkdir -p .risingwave/config
+  cat <<EOF > .risingwave/config/risedev-env
+RW_META_ADDR="http://127.0.0.1:5690"
+RISEDEV_RW_FRONTEND_LISTEN_ADDRESS="127.0.0.1"
+RISEDEV_RW_FRONTEND_PORT="4566"
+EOF
```

Comment:
I would suggest doing this in the `playground` binary so that developers can also benefit from it locally. This is to address https://github.com/risingwavelabs/risingwave/issues/17090. Can do it in next PRs.

Reply:
Good idea.

Reply:
https://github.com/risingwavelabs/risingwave/issues/17700